### PR TITLE
Introduce new orbit bundle for cdc inbound endpoint in MI.

### PR DESCRIPTION
**Pull request by RusJaI**

### CODE REVIEW

**Comment by lasanthaS** [comment]
@RusJaI Shall we rename the component as `debezium-2.1.4..wso2v1` instead? So the pom.xml path would be `debezium/2.1.4..wso2v1/pom.xml`.

File: cdc-mi/1.0.0wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,248 @@
+<!--
```
Comment (lasanthaS) [inline cdc-mi/1.0.0wso2v1/pom.xml:1]: Component path change as mentioned in https://github.com/wso2/orbit/pull/995#issuecomment-1670613314

**Review by lasanthaS — CHANGES_REQUESTED** [review]

File: cdc-mi/1.0.0wso2v1/pom.xml (hunk 1 above)
Comment (RusJaI) [reply cdc-mi/1.0.0wso2v1/pom.xml:1]: is it okay to name it as Debezium since it contains Kafka and binlog connector dependencies as well?

Comment (lasanthaS) [reply cdc-mi/1.0.0wso2v1/pom.xml:1]: Yeah, that's fine. We are already doing similar in some of the other orbit bundles as well.

Comment (RusJaI) [reply cdc-mi/1.0.0wso2v1/pom.xml]: Thanks for the clarification.  Changed the name accordingly

**Review by lasanthaS — APPROVED** [review]

**Pull request merged**